### PR TITLE
[docs] Edits to the multi-window mode docs requested by @miherlosev

### PR DESCRIPTION
**Pull request by titerman**
See https://github.com/DevExpress/testcafe/pull/5400

### CODE REVIEW

File: docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md [hunk 1]
```diff
@@ -18,16 +18,20 @@ When your page launches a new window, the test automatically continues in the ne
...
-    await t
-        .click(googleButton)
-        .typeText('input[type=email]', 'This text will be entered inside the login dialog');
+fixture `Microsoft DevBlogs Login page`
+    .page('https://devblogs.microsoft.com/');
+
+test('automatically change execution context', async t => {
+   await t
+    .click('.login-but') #click the login button
+    .click('.nsl-button-google') #open Google's OAuth window
```
Comment (miherlosev) [inline docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md:27]: Need to use the JavaScript comment  here and below
`// open Google's OAuth window`

File: docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ permalink: /documentation/guides/advanced-guides/multiple-browser-windows.html
...
 
-⚠ This is a **beta** feature. Browser support is limited to local instances of Chrome and Firefox. You cannot capture videos or screenshots of child windows. The available functionality is subject to further revisions. Please do not use this feature in production environments.
+⚠ This is a **beta** feature. Browser support is limited to local instances of Chrome and Firefox. You cannot resize child windows. You cannot take screenshots or videos of child windows. The available functionality is subject to further revisions. Please do not use this feature in production environments.
 
-️🛠️ Use the `--disable-multiple-windows` CLI flag to disable support for multiple browser windows if you encounter compatibility issues with [TestCafe v2015.1](http://testcafe.devexpress.com/documentation) tests.
+️🛠️ Use the `--disable-multiple-windows` CLI flag to disable support for multiple browser windows if you encounter compatibility issues with your existing tests.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md:12]:
```suggestion
️🛠️ Use the `--disable-multiple-windows` CLI flag to disable support for multiple browser windows if you encounter compatibility issues with your tests.
```

File: docs/articles/documentation/reference/test-api/testcontroller/openwindow.md [hunk 3]
```diff
@@ -6,15 +6,15 @@ permalink: /documentation/reference/test-api/testcontroller/openwindow.html
 
 # t.openWindow method
 
-Opens a new browser window. Returns the matching window descriptor.
+Opens a new browser window. Returns the matching window descriptor, but can still be chained with other TestController methods.
```
Comment (dirk-pieterse) [inline docs/articles/documentation/reference/test-api/testcontroller/openwindow.md:9]:
```suggestion
Opens a new browser window. Returns the matching window descriptor and can be chained with other TestController methods.
```

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot /retest

**Comment by titerman** [comment]
@testcafe-build-bot /retest

**Pull request merged**